CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [3.0.0]
+
+- This is first version using Bitmovin Player iOS V4 SDK. This is not backward compatible with previous versions of the player SDK.
```

Comment:
Is this entry needed? Also the main change in this update is new version of Conviva SDK and there is no change in Bitmovin player SDK.